foundations/tying_it_all_together/conclusion.md

```diff
@@ -42,7 +42,7 @@ To help you make a decision, here are some perks of each technology:
 
 In the end, it doesn't matter which path you take. It's only important that you stick with it, and continue to hold onto the learner's mindset throughout your career. Not convinced? [Click here](https://medium.com/@bycdiaz/choosing-the-right-language-a-short-guide-on-how-not-to-ruin-your-career-2b353be1371).
 
-### Still Can’t Decide, Let the Spinner Settle Your Fate
+### Still can’t decide, let the spinner settle your fate
```

Comment:
```suggestion
### Still can’t decide? let the spinner settle your fate!
```

I thought this would make more sense than the comma, what do you think?

Reply:
I like this suggestion with one minor tweak that I will push with my other fixes. (cap Let, since new sentence)

### Still can’t decide? Let the spinner settle your fate!

Reply:
Oh yes! Perfect catch, thank you!